src/main/java/fruitshop/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,20 @@
+package fruitshop.strategy;
+
+import fruitshop.db.Storage;
+import fruitshop.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    private final Storage storage = new Storage();
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the start of the class definition.